modules/unsupported/tile-client/src/test/java/resources/logging.properties

```diff
@@ -0,0 +1,5 @@
+java.util.logging.ConsoleHandler.level = FINE
+java.util.logging.ConsoleHandler.formatter = java.util.logging.SimpleFormatter
+java.util.logging.ConsoleHandler.encoding = Cp850
+
+org.geotools.tile = FINE
```

Comment:
Not sure if you intended to commit this, we generally leave logging turned down during builds.